src/subgraph_endpoints.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
1) these names should be in the form: http[networkname] and  and ws[NetworkName]

2) "mainnet" should be "main"

so like:

http_main, ws_main
rinkeby_main, ws_,main

etc

Reply:
why does it matter ?

Reply:
1) this naming pattern is standard best practice in coding.

2) main is that we're using almost everywhere in Alchemy, including in settings.ts